### PR TITLE
fix: wallet module spaming epochs

### LINKED ISSUE #3202: Wallet module spamming epochs on zero height or something
The reason for https://github.com/fedimint/fedimint/issues/3102 is that wallet modules is spamming with epochs (generating them endlessly) for some reason. If I can get   https://github.com/fedimint/fedimint/pull/3201 to fail i'll be able to debug. :D

### PR DESCRIPTION
**Pull request by dpc**
Fix #3202

### CODE REVIEW

File: modules/fedimint-wallet-common/src/config.rs [hunk 1]
```diff
@@ -61,6 +61,8 @@ pub struct WalletConfig {
 pub struct WalletConfigLocal {
     /// Configures which bitcoin RPC to use
     pub bitcoin_rpc: BitcoinRpcConfig,
+    /// Our own peer_id
+    pub our_peer_id: PeerId,
```
Comment (dpc) [inline modules/fedimint-wallet-common/src/config.rs:65]: I don't know if we should put it here, or is there's a better way.

File: modules/fedimint-wallet-server/src/lib.rs [hunk 2]
```diff
@@ -325,16 +327,19 @@ impl ServerModule for Wallet {
         // TODO: We should not be panicking
         let block_count = self.get_block_count().await.expect("bitcoind rpc failed");
         let block_count_proposal = block_count.saturating_sub(self.cfg.consensus.finality_delay);
-        let consensus_block_count = self.consensus_block_count(dbtx).await;
```
Comment (dpc) [inline modules/fedimint-wallet-server/src/lib.rs:328]: Observation: we don't care what's the consensus here. Our job is to submit our vote, and our previous vote is all that concerns us.

**Review by douglaz — APPROVED** [review]
I don't know if there is a better way, but LGTM

File: modules/fedimint-wallet-common/src/config.rs (hunk 1 above)
Comment (elsirion) [reply modules/fedimint-wallet-common/src/config.rs:65]: Maybe this should be a parameter of `ServerModuleInit::init` instead?

https://github.com/fedimint/fedimint/blob/6c15b0ea133b5239085884c14c334623287d555b/fedimint-core/src/module/mod.rs#L583-L588

Comment (dpc) [reply modules/fedimint-wallet-common/src/config.rs:65]: Maybe. Would be easier if Re https://github.com/fedimint/fedimint/issues/3025 was ready. :D

Comment (dpc) [reply modules/fedimint-wallet-common/src/config.rs:65]: Why not both? :shrug: :D

**Pull request merged**